app/menus/menus/window.js

```diff
@@ -1,15 +1,14 @@
-const {accelerators} = require('../../accelerators');
-
-module.exports = function () {
+module.exports = function (commands) {
   return {
     role: 'window',
     submenu: [
       {
         role: 'minimize',
-        accelerator: accelerators.minimize
+        accelerator: commands['window:minimize']
       },
       {
-        role: 'zoom'
+        role: 'zoom',
```

Comment:
Alternately, that could be a bug, It looks like that is supposed to be the same as "Maximize" in windows and linux, which don't appear anywhere in the menus on either (though they do appear in the window controls).